pages/post.js

```diff
@@ -147,6 +147,21 @@ const personComponent = ({
   );
 };
 
+const renderNoFollowLinks = (children) => children.reduce((acc, item) => {
+  if (typeof item === 'object' && item.type === 'a') {
+    const { props: { children: child, href } } = item;
+    const isNofollowLink = blogPostsNoFollowLinks.includes(href);
+    if (isNofollowLink) {
+      acc.push(<a rel="nofollow noopener" href={href}>{child}</a>);
+    } else {
+      acc.push(<a href={href}>{child}</a>);
+    }
```

Comment:
@stefan9541 ,

1. Could this if-else be shortened to one-lliner like this?

```js
acc.push(<a rel={isNofollowLink ? "nofollow noopener" : ""} href={href}>{child}</a>);
```

2. Rather than using `acc.push` would you be able to use a declarative approach like this:

```js
return [...acc, link]
```

3. Instead of outer if-else statement, could you use "guard-statements"? E.g.:

```js
if (typeof item !== 'object' || item.type !== 'a') {
  return [...acc, item]
}

const { props: { children: child, href } } = item;
const isNofollowLink = blogPostsNoFollowLinks.includes(href);
return [
  ...acc,
  <a rel={isNofollowLink ? "nofollow noopener" : ""} href={href}>{child}</a>
];
```

Reply:
@epaminond check again, pls